### PR TITLE
Add pip reinstall

### PR DESCRIPTION
**Pull request by nghi-ly**
## What are you changing in this pull request and why?

Try --force-reinstall  parameter if having trouble installing dbt Cloud CLI

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [x] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [x] Needs review from PM

### CODE REVIEW

**Comment by greg-mckeon** [comment]
lgtm!

**Review by greg-mckeon — APPROVED** [review]

File: website/docs/docs/cloud/cloud-cli-installation.md [hunk 1]
```diff
@@ -205,6 +205,11 @@ We recommend using virtual environments (venv) to namespace `cloud-cli`.
   pip install dbt --no-cache-dir
   ```
 
+  If there are installation issues, try running the command with the `--force-reinstall` argument:
```
Comment (mirnawong1) [inline website/docs/docs/cloud/cloud-cli-installation.md]: sugg concise a bit more concise
```suggestion
  If there are installation issues, running the command with the `--force-reinstall` argument might help:
```

**Review by mirnawong1 — APPROVED** [review]
🚢

File: website/docs/docs/cloud/cloud-cli-installation.md (hunk 1 above)
Comment (nghi-ly) [reply website/docs/docs/cloud/cloud-cli-installation.md]: i wanted to convey that it's possible that even with the --arg their install issue still might not go away, which the sugg doesn't quite capture imho. i was torn between these two and went with the first for conciseness:

- "... try running the command with ..."
- "... running the command with ... might help"

sounds like i should use the second one for the sake of clarity.

**Pull request merged**